### ISSUE TITLE
Add option to hide title block

### ISSUE THREAD

**Issue by wesleybl** [plone/volto#4935]
**Is your feature request related to a problem? Please describe.**
For layout reasons, we need an option to not show the title of a content. For accessibility reasons, we cannot remove the title block, as every page needs one. So we should have an option to hide with CSS.

**Describe the solution you'd like**
It may be necessary to create a behavior with a "Hide title" field. Or would the option on the block itself be better?

**Comment by davisagli** [comment]
The title can still be edited in the sidebar even if there isn't a title block, so I don't understand how a title hidden with CSS is helpful for accessibility.

**Comment by wesleybl** [comment]
@davisagli some accessibility validators say that the page must have an h1 (title block in the case of Volto). To see:

https://dequeuniversity.com/rules/axe/4.1/page-has-heading-one

At first, I thought this was an accessibility error. But later I saw that it is only a best practice recommendation, made by the validators, not an error.

So I think it's not worth implementing this issue, since it's not an error.

Closing here.
